### PR TITLE
Add support for PCSC socket

### LINKED ISSUE #2777: Add PCSC socket
To allow multiple access to smart cards or security keys, there is a daemon called `pcscd` that has a socket available at `/run/pcscd/pcscd.comm` or the value of environment variable `PCSCLITE_CSOCK_NAME`.

Some application may need that, for example Yubico applications for their keys.

### PR DESCRIPTION
**Pull request by valentindavid**
Fixes #2777

### CODE REVIEW

File: common/flatpak-run.c [hunk 1]
```diff
@@ -284,6 +284,33 @@ flatpak_run_add_ssh_args (FlatpakBwrap *bwrap)
...
 }
 
+static void
+flatpak_run_add_pcsc_args (FlatpakBwrap *bwrap)
+{
+  const char * pcsc_socket;
+  const char * sandbox_pcsc_socket = "/run/pcscd/pcscd.comm";
+
+  pcsc_socket = g_getenv ("PCSCLITE_CSOCK_NAME");
+  if (pcsc_socket) {
```
Comment (TingPing) [inline common/flatpak-run.c]: wrong whitespace/formatting.

**Comment by alexlarsson** [comment]
bot, add author to whitelist

File: common/flatpak-run.c (hunk 1 above)
Comment (alexlarsson) [reply common/flatpak-run.c]: Yeah, brackets on a new line, etc.

**Comment by alexlarsson** [comment]
Docs need updating too, check all places that mention e.g. `ssh-auth`

**Comment by alexlarsson** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**